tests/integration-tests/tests/storage/test_efs.py

```diff
@@ -120,23 +130,25 @@ def test_multiple_efs(
 
 
 @pytest.fixture(scope="class")
-def efs_stack_factory(cfn_stacks_factory, request, region):
+def efs_stack_factory(cfn_stacks_factory, request, region, vpc_stack):
     """EFS stack contains a single efs resource."""
     created_stacks = []
 
-    def create_efs():
+    def create_efs(num=1, create_mount_targets=False):
```

Comment:
`create_mount_targets` doesn't seem to be used in this function unless I'm missing something

Reply:
You are right! Done